EasyLearn/app/helpers/player_helper.rb

```diff
@@ -0,0 +1,2 @@
+module PlayerHelper
```

Comment:
If you don't need this file, you can delete it

Reply:
Deleted